packages/developer-console-spa/src/components/ConfigureLighthouse.tsx

```diff
@@ -0,0 +1,237 @@
+import LinkProjectForm from './lighthouse/LinkProjectForm';
+import { AppContext } from '../context/AppContext';
+import { getLHSpaConfigByAppId } from '../services/lighthouse';
+
+const ConfigureLighthouse = ( props: any ) => {
+  const [ isModalOpen, setIsModalOpen ] = useState( false );
+  const [ activeTabKey, setActiveTabKey ] = useState( 0 );
+  const [ selectedProject, setSelectedProject ] = useState( {
+    name: '',
+    id: '',
+    adminToken: '',
+    token: ''
+  });
+  const [ isCardExpanded, setIsCardExpanded ] = useState( true );
+  const [ showConfirmation, setShowConfirmation ] = useState( false );
+  const { appId, app } = useContext( AppContext );
+  const [ isProjectLinked, setIsProjectLinked ] = useState( false );
+
+  useEffect( () => {
+    getLHSpaConfigByAppId( app.id ).then( ( res: any ) => {
+      if ( res ) {
+        setIsProjectLinked( true );
+      }
+    })
+  }, []);
+  // Modal helpers
+  const handleModalToggle = () => {
+    setIsModalOpen( !isModalOpen );
+    setShowConfirmation(false);
+  }
+  const handleTabClick = ( event: React.MouseEvent<HTMLElement, MouseEvent>, tabIndex: string | number ) => {
+    setActiveTabKey(tabIndex as number);
+  }
+  const openLighthouseDoc = () => {
+    window.open('https://github.com/GoogleChrome/lighthouse-ci/blob/main/docs/getting-started.md', '_blank',);
+  };
+  const openLighthouseConfigDoc = () => {
+    window.open( 'https://github.com/GoogleChrome/lighthouse-ci/blob/main/docs/configuration.md', '_blank', );
+  };
+  const openOPLighthouseDoc = () => {
+    window.open('/get-started/docs/apps/internal/lighthouse', '_blank',);
+  };
+
+  const onCardExpand = () => {
+    setIsCardExpanded( !isCardExpanded );
+  };
+
+  return (
+    <>
+      <Stack hasGutter>
+        <StackItem>
+          <Header title="Configure Lighthouse" />
+        </StackItem>
+        { !isProjectLinked ?
+          <StackItem>
+            <Card isRounded>
+              <CardBody>
+                <EmptyState>
+                  <EmptyStateBody>
+                    <img src={ lhIllustration } alt="Lighthouse Illustration" />
+                  </EmptyStateBody>
+                  <Button onClick={ handleModalToggle } variant="primary">Get Started</Button>
+                  <EmptyStateSecondaryActions>
+                    <Button variant="link" onClick={ openOPLighthouseDoc }>Learn more</Button>
+                  </EmptyStateSecondaryActions>
+                </EmptyState>
+              </CardBody>
+            </Card>
+          </StackItem>
+        :
+          <StackItem>
+            <Card>
+              <CardHeader>
+                <CardActions>
+                  <Button
+                    aria-label="Delete Branch"
+                    alt="Delete Branch" variant="link"
+                  >
+                    <TrashIcon />
+                  </Button>
+                </CardActions>
+                <CardTitle>
+                  <CheckCircleIcon /> You are all set
```

Comment:
Follow the designs and change the styles for this message.

Also, it does not have to be in the CardTitle as it has fixed styles, you can just put it in the CardBody and add custom styles to it.